frontend/src/components/managePermissions/UserDataTable.vue

```diff
@@ -1,15 +1,19 @@
 <script setup lang="ts">
 import { reactive, ref } from 'vue';
-import router from '@/router';
-import IconCapitol from '@/components/common/IconCapitol.vue';
+import type { PropType } from 'vue';
+
 import { FilterMatchMode } from 'primevue/api';
 import Column from 'primevue/column';
 import DataTable from 'primevue/datatable';
 import InputText from 'primevue/inputtext';
-import { IconSize } from '@/enum/IconEnum';
 import { useConfirm } from 'primevue/useconfirm';
 import ConfirmDialog from 'primevue/confirmdialog';
-import type { PropType } from 'vue';
+
+import router from '@/router';
+import { IconSize } from '@/enum/IconEnum';
+import IconCapitol from '@/components/common/IconCapitol.vue';
+import Button from '@/components/common/Button.vue';
```

Comment:
a lot of changes in this file are just auto formatting, the real changes mainly at the end of the file about styling